docs/reference/commandline/service_create.md

```diff
@@ -177,6 +177,45 @@ $ docker service create \
   redis:3.0.6
 ```
 
+### Container networking
+The containers provisioned in docker swarm mode can be accessed in service discovery either via a Virtual IP (VIP) and routed through the docker swarm ingress overlay network. Or via a DNS round robbin (DNSRR). This configuration is set using the `--endpoint-mode [vip|dnsrr]` setting. Note that dnsrr is incompatible with `--published` ports.
```

Comment:
@mrjana , @sanimej , can you confirm whether or not DNSRR is compatible with `--published` ports or not?